docs/release/role-handbooks/communications/README.md

```diff
@@ -0,0 +1,191 @@
+
+Welcome to the Cluster API Comms Team onboarding documentation!
+
+## Overview
+
+- Familiarize yourself with the release process, following the order of priority:
+  - Reference the [release timeline](../../releases/) of the release cycle you are involved to understand the release process.s  - Walk through the [release note generation process](#create-pr-for-release-notes) and try to generate notes by yourself.
+  - Understand the release notes tool [code](https://github.com/kubernetes-sigs/cluster-api/tree/main/hack/tools/release). The release team is in charge of maintaining this tool and updating it to cover new cases or add new features.
+- Documentation familiarity::  - Explore project's [documentation](https://cluster-api.sigs.k8s.io/introduction) and start learning how to update and maintain it.
+- Communication channels:
+  - Learn about project's primary communication channels, including Slack, mailing list and office hours. Try to understand how information is distributed and how to use them effectively for announcements, updates and discussions related to the release cycle.
+
+## Release Tasks
+This document details the responsibilities and tasks for release lead team in the release process.
+
+   - [Responsibilities](#responsibilities)
+  - [Tasks](#tasks)
+    - [Add docs to collect release notes for users and migration notes for provider implementers](#add-docs-to-collect-release-notes-for-users-and-migration-notes-for-provider-implementers)
+    - [Update supported versions](#update-supported-versions)
+    - [Ensure the book for the new release is available](#ensure-the-book-for-the-new-release-is-available)
+    - [Generate weekly PR updates to post in Slack](#generate-weekly-pr-updates-to-post-in-slack)
+    - [Create PR for release notes](#create-pr-for-release-notes)
+    - [Change production branch in Netlify to the new release branch](#change-production-branch-in-netlify-to-the-new-release-branch)
+    - [Update clusterctl links in the quickstart](#update-clusterctl-links-in-the-quickstart)
+    - [Continuously: Communicate key dates to the community](#continuously-communicate-key-dates-to-the-community)
+    - [Communicate beta to providers](#communicate-beta-to-providers)
+
+❗Notes:his document are based on the v1.6 release cycle.
+- This document focuses on tasks that are done for every release. One-time improvement tasks are out of scope.
+- If a task is prefixed with [Track] it means it should be ensured that this task is done, but the folks with the corresponding role are not responsible to do it themselves.
+
+
+## Responsibilities
+
+- Communication:
+  - Communicate key d  - Communicate key dates to the community
+- Documentation:r  - Improve release process documentation
+  - Ensure the book and provider upgrade documentation are up-to-date
+  - Maintain and improve user facing documentation about releases, release policy and release calendar
+- Release Notes:s  - Create PR with release notes
```

Comment:
this looks like it got chopped up, should adhere to the original formatting: https://github.com/kubernetes-sigs/cluster-api/blob/af7ef298810de29ade959f4252087069eb49ae5d/docs/release/release-tasks.md?plain=1#L301-L308